datastore/go.mod

```diff
@@ -0,0 +1,7 @@
+module google.golang.org/appengine
```

Comment:
remove this file? there's already a go.mod in the root

Reply:
done